### DISCUSSION TITLE
pointcolrow factor uitfaseren

### DISCUSSION THREAD

**Discussion by jipclaassens** [ObjectVision/GeoDMS#46] [Ideas]
- waar speelt dit allemaal?
 - fucnties als pointcol/pointrow/point

**Comment by jipclaassens** [comment]
gerelateerd #136

**Comment by jipclaassens** [comment]
Gerelateerd?

Mantis#335 https://geodms.nl/mantis/view.php?id=335

**Comment by MaartenHilferink** [comment]
In 18.1.2 speelt dit nog bij de Range = "{(bottom, left), (top, right)". In 18.1.3 geeft deze contruction een obsolete cfg point order error
